src/twisted/python/runtime.py

```diff
@@ -203,4 +208,3 @@ def supportsINotify(self) -> bool:
 
 platform = Platform()
 platformType = platform.getType()
-seconds = platform.seconds
```

Comment:
seconds are no longer special cased on any platform

Reply:
understood...but I think that this needs to be deprecated first...and removed later.

Reply:
or revert the change and remove it and deprecated it in a separate PR

Reply:
ok. stupid me. `from time import time as seconds`

Reply:
This whole file needs deprecating from orbit